ksml/src/test/java/io/axual/ksml/TopologyGeneratorBasicTest.java

```diff
@@ -91,6 +91,19 @@ void cleanDescriptionTestLineEnd() {
         assertThat(cleanDescription("fix\r\nnewlines"), is("fix\nnewlines"));
     }
 
+    @Test
+    void loadGeneratorExample() throws Exception {
```

Comment:
Can you clarify what is tested here?
As I see it it tests that the file can be successfully parsed to a topology?
Can we rename the test to something more descriptive, maybe without the example in the name?

Reply:
Hi Richard, the intention was to have a test proving that `interval:` can be optional, this is now in `IntervalScheduleTest`. Removed the test and the resource; also made the test conditional on GraalVM (`@EnabledIf`) instead of the ugly fail if not Graal.